test/Elastica/ResponseFunctionalTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Elastica\Test;
+
+use Elastica\Document;
+use Elastica\Exception\ResponseException;
+use Elastica\Mapping;
+use Elastica\Query;
+use Elastica\Query\MatchAll;
+use Elastica\Request;
+use Elastica\Test\Base as BaseTest;
+
+/**
+ * @group functional
+ */
+class ResponseFunctionalTest extends BaseTest
```

Comment:
Always happy to have more tests. Could you share some background if this is "in addition" or replacing something else?

Reply:
Those are tests extracted from the ResponseTest